backend/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "target": "ESNext", 
+    "module": "ESNext",
+    "moduleResolution": "Node",
+    "esModuleInterop": true,
+    "strict": true,
+    "skipLibCheck": true,
+    "outDir": "./dist",
+  },
+  "include": ["actions", "server.ts"]
```

Comment:
If we add another folder under /backend, we will need to remember to add it to this array.

Maybe we should simply include all files in the /backend folder with this pattern: `"include": ["./**/*"]`